### PR TITLE
Application workflow prototype: tweaks based on feedback

### PR DESCRIPTION
**Pull request by benilovj**
### Changes proposed in this pull request

Tweaks in response to feedback left on #172

### CODE REVIEW

File: features/apply_1.feature [hunk 1]
```diff
@@ -40,5 +40,4 @@ Feature: Apply 1
   Examples:
     | # of previous applications |
     | 1                          |
-    | 2                          |
     | 3                          |
```
Comment (duncanjbrown) [inline features/apply_1.feature]: And 3?

**Comment by duncanjbrown** [comment]
There are a couple more things to add on here following comments on #172

File: features/apply_1.feature (hunk 1 above)
Comment (benilovj) [reply features/apply_1.feature]: I think that 3 should stay in, to show that 1 isn't some kind of magic number.

**Comment by benilovj** [comment]
@duncanjbrown could we please merge this as is and I will cover off the remaining comments on #172 on a separate PR? I'm keen to reduce work in progress, and merging these feels like an improvement.

**Pull request merged**